cumulus_library/studies/core/reference_sql/builder_encounter.sql

```diff
@@ -582,7 +582,8 @@ temp_encounter_completion AS (
         SELECT
             ece.encounter_id,
             (
-                BOOL_OR(ec.table_name = 'condition')
+                BOOL_OR(ec.table_name = 'allergyintolerance')
+                AND BOOL_OR(ec.table_name = 'condition')
```

Comment:
This is from a previous PR where I forgot to regenerate SQL after making a change.

Since the current flow is to generate SQL from test data, we could add a check in the CI to regenerate SQL and see if there are any changes - failing the PR if so... Just a thought for later

Reply:
#182 - it's been hanging around for a bit, but it remains a good idea.